page/PatientInformation.jsx

```diff
@@ -0,0 +1,60 @@
+import { View, Text, StyleSheet } from 'react-native';
+import React from 'react';
+import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view';
+import useFetchDataById from '../hooks/useFetchDataById';
+import useFetch from '../hooks/useFetch';
+
+const PatientInformation = () => {
+	const { getPatientsFilter } = useFetch();
+	const { data } = useFetchDataById(getPatientsFilter);
+	return data.map(patient => (
+		<KeyboardAwareScrollView>
+			<View>
+				<Text style={styles.labelTittle}>Informacion del Paciente:</Text>
+				<View style={styles.lineStyle} />
+				<Text style={styles.labelSubTittle}>Nombre:</Text>
+				<Text style={styles.patientInfo}>{patient.name}</Text>
+				<View style={styles.lineStyle} />
+				<Text style={styles.labelSubTittle}>Edad:</Text>
+				<Text style={styles.patientInfo}>13</Text>
+				<View style={styles.lineStyle} />
+				<Text style={styles.labelSubTittle}>Dx Oncologicó:</Text>
+				<Text style={styles.patientInfo}>si</Text>
+				<View style={styles.lineStyle} />
+				<Text style={styles.labelSubTittle}>Paliativo:</Text>
+				<Text style={styles.patientInfo}>si</Text>
+				<View style={styles.lineStyle} />
+				<Text style={styles.labelSubTittle}>Enfermera:</Text>
+				<Text style={styles.patientInfo}>Erika Valverde</Text>
+				<View style={styles.lineStyle} />
```

Comment:
We should use English only

Reply:
Also, why aren't you taking the `nurse's name`, `palliative` and other values from the `patient` variable?